<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EasyPicker</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/new/Icons/icons8-circuit-80.png</normaloff>:/new/Icons/icons8-circuit-80.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="cameraComboBox"/>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QWidget" name="cameraView" native="true">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Head Location</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Part Name:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="partNameLabel">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Head Location:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="headLocationLabel">
               <property name="text">
                <string>(0,0,0)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>Home</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Pick Up Location:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="pickUpLocationLabel">
               <property name="text">
                <string>(0,0,0)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>Set Pick Up</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Increment (mm):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_2">
               <item>
                <property name="text">
                 <string>0.1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>0.5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>3</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>10</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>25</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>50</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="minimumSize">
              <size>
               <width>210</width>
               <height>210</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>200</height>
              </size>
             </property>
             <property name="title">
              <string>Adjust X/Y Location</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="1" column="1">
               <widget class="QPushButton" name="adjustTopPushButton">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="Icons.qrc">
                  <normaloff>:/new/Icons/up-100.png</normaloff>:/new/Icons/up-100.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>50</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="shortcut">
                 <string>Up</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QPushButton" name="adjustBottomPushButton">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="Icons.qrc">
                  <normaloff>:/new/Icons/down-100.png</normaloff>:/new/Icons/down-100.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>50</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="shortcut">
                 <string>Down</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QPushButton" name="adjustLeftPushButton">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="Icons.qrc">
                  <normaloff>:/new/Icons/left-100.png</normaloff>:/new/Icons/left-100.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>50</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="shortcut">
                 <string>Left</string>
                </property>
                <property name="default">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QPushButton" name="adjustRightPushButton">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="Icons.qrc">
                  <normaloff>:/new/Icons/right-100.png</normaloff>:/new/Icons/right-100.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>50</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="shortcut">
                 <string>Right</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>716</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Project"/>
   <addaction name="actionOpen"/>
   <addaction name="actionImport_CSV"/>
   <addaction name="separator"/>
   <addaction name="actionStart_Automated"/>
   <addaction name="actionStep_Through_Next"/>
   <addaction name="actionStep_Through_Back"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
  </widget>
  <action name="actionNew_Project">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/new/Icons/icons8-new-80.png</normaloff>:/new/Icons/icons8-new-80.png</iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Create a New Pick and Place Project.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;This will close the existing project.  &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;Any unsaved changes could be lost!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/new/Icons/icons8-open-box-80.png</normaloff>:/new/Icons/icons8-open-box-80.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open a project</string>
   </property>
  </action>
  <action name="actionImport_CSV">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/new/Icons/icons8-import-csv-80.png</normaloff>:/new/Icons/icons8-import-csv-80.png</iconset>
   </property>
   <property name="text">
    <string>Import CSV</string>
   </property>
   <property name="toolTip">
    <string>Import a CSV File to the active Project</string>
   </property>
  </action>
  <action name="actionStart_Automated">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/new/Icons/icons8-launch-80.png</normaloff>:/new/Icons/icons8-launch-80.png</iconset>
   </property>
   <property name="text">
    <string>Start Automated</string>
   </property>
   <property name="toolTip">
    <string>Start Automated Pick and Place</string>
   </property>
  </action>
  <action name="actionStep_Through_Next">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/new/Icons/icons8-obstruction-lights-on-80.png</normaloff>:/new/Icons/icons8-obstruction-lights-on-80.png</iconset>
   </property>
   <property name="text">
    <string>Step Through Next</string>
   </property>
  </action>
  <action name="actionStep_Through_Back">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/new/Icons/icons8-obstruction-lights-settings-80.png</normaloff>:/new/Icons/icons8-obstruction-lights-settings-80.png</iconset>
   </property>
   <property name="text">
    <string>Step Through Back</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/new/Icons/icons8-settings-80.png</normaloff>:/new/Icons/icons8-settings-80.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/new/Icons/icons8-manual-80.png</normaloff>:/new/Icons/icons8-manual-80.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
